demos/python_demos/clip_image_classification/client.py

```diff
@@ -16,16 +16,22 @@
 import sys
 sys.path.append("../../common/python")
 import tritonclient.grpc as grpcclient
+from tritonclient.grpc import service_pb2
+from tritonclient.grpc import service_pb2_grpc
 import argparse
 import datetime
 import numpy as np
 from client_utils import print_statistics
 from urllib.request import urlretrieve
 from pathlib import Path
 import os
+import grpc
+import time
 
 parser = argparse.ArgumentParser(description='Client for clip example')
 
+parser.add_argument('--timeout', required=False, default='15',
```

Comment:
why this parameter was added? timeout usually describe how long we wait for the response, not the readiness.